### DISCUSSION TITLE
Thanks

### DISCUSSION THREAD

**Discussion by Typhooteaman1** [sircharlo/meeting-media-manager#281] [General]
Just wanted to say used the new version that works with hybrid meetings today for first time showing the videos and pictures and streamed into zoom, worked so well, just wanted to say thank you for the hard work that has gone into making this such a useful tool for the meetings!!

**Comment by sircharlo** [comment]
Always a pleasure. I'm just glad some can make use of it!

**Comment by Typhooteaman1** [comment]
We appreciate your hard work on the programming. We were having lots of
problems with JW library differently not planning the sound when zoom was
sharing, or often we had to turn the second screen off and bacj on again in
JW library to get the display working again on the monitors, but it all
worked perfectly when we used this program 😁. Took a lot of stress out of
running the hybrid meeting .

Your brother Rob
